packages/base/src/3dview/mainview.tsx

```diff
@@ -686,6 +716,14 @@ export class MainView extends React.Component<IProps, IStates> {
     });
   }
 
+  private _handleSettingsChange(): void {
+    if (this._sceneAxe) {
+      this._sceneAxe.visible = this._model.jcadSettings().showAxesHelper;
+    }
+
+    this._updateCamera();
```

Comment:
you are updating everything on any setting change

Reply:
Updated it to respect the changed key